[2021-03-22 21:15:05,946] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 2021-03-20T00:00:00+00:00 [queued]>
[2021-03-22 21:15:05,968] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 2021-03-20T00:00:00+00:00 [queued]>
[2021-03-22 21:15:05,968] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-22 21:15:05,969] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-22 21:15:05,969] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-22 21:15:05,985] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): runme_0> on 2021-03-20T00:00:00+00:00
[2021-03-22 21:15:05,992] {standard_task_runner.py:52} INFO - Started process 55 to run task
[2021-03-22 21:15:05,998] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_0', '2021-03-20T00:00:00+00:00', '--job-id', '5', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpy4g39bro', '--error-file', '/tmp/tmpd6sh04qj']
[2021-03-22 21:15:05,999] {standard_task_runner.py:77} INFO - Job 5: Subtask runme_0
[2021-03-22 21:15:06,060] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_0 2021-03-20T00:00:00+00:00 [running]> on host 699fbf69fb73
[2021-03-22 21:15:06,110] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_0
AIRFLOW_CTX_EXECUTION_DATE=2021-03-20T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-20T00:00:00+00:00
[2021-03-22 21:15:06,110] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-03-22 21:15:06,111] {bash.py:158} INFO - Running command: echo "example_bash_operator__runme_0__20210320" && sleep 1
[2021-03-22 21:15:06,119] {bash.py:169} INFO - Output:
[2021-03-22 21:15:06,120] {bash.py:173} INFO - example_bash_operator__runme_0__20210320
[2021-03-22 21:15:07,122] {bash.py:177} INFO - Command exited with return code 0
[2021-03-22 21:15:07,145] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_0, execution_date=20210320T000000, start_date=20210322T211505, end_date=20210322T211507
[2021-03-22 21:15:07,173] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-22 21:15:07,213] {local_task_job.py:146} INFO - Task exited with return code 0
